### DISCUSSION TITLE
NES CLASSIC MINI

### DISCUSSION THREAD

**Discussion by Wences01** [darthcloud/BlueRetro#296] [Q&A] [answered]
Can Blueretro be installed on the Mini Nes Classic console?

What are the previous settings?

Thank you! I love your work!

**Comment by darthcloud** [comment, marked as answer]
Possible if you are not using Hakchi.

I don't have one so untested and you are on your own to try to adapt the SNES mini instruction:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-SNES-Mini-Internal-Install

**Reply by fermel01** [reply]
For You Information:
https://github.com/TeamShinkansen/Hakchi2-CE/issues/426#issuecomment-1155494159
